src/composables/use-focus-on-show.ts

```diff
@@ -14,7 +14,7 @@ type Props = {
   dialogRef: Ref<HTMLElement | null>
   visibleRef: Ref<boolean>
   autoFocusOnShowRef: Ref<boolean>
-  initialFocusElementRef: Ref<HTMLElement | null>
+  initialFocusElementRef?: Ref<HTMLElement | null>
```

Comment:
Having a value that can be both `null` and `undefined` can be confusing.

Reply:
When a prop is not required in a Vue component, it can be undefined. So, without the `?` Typescript raises an error.